kuma/urls.py

```diff
@@ -105,6 +102,9 @@
     ),
 )
 urlpatterns += i18n_patterns(re_path(r"^payments/", include(payments_lang_urlpatterns)))
+urlpatterns += i18n_patterns(
+    re_path(r"payments$", payment_views.index, name="payments")
+)
```

Comment:
I think we need to think deeper about this. It's nice that we're being consistent and not using a trailing slash if other pages don't use that. However, `/payments/` is actually a namespace. Look at the line above (line 104). 
So what you have name is a clash. 
Basically, I think the `payments_view.index` needs to be moved into `payments/urls.py`.

Reply:
Will work on this in the next few minutes 👍

Reply:
This is now fixed-- the index page is now being handled in `payments/urls.py`. Thanks for the suggestion!